script.js

```diff
@@ -1,4 +1,135 @@
+// Project 2 : Beat that (Base)//
+
+// 1. Number of players = 2 , will take turns to play the game
+// Create a prompt to inform player to click on submit button.
+
+// 2. When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6. The dice values should be stored for each player.
+
+// 3. The player picks the order of the dice they want. (e.g whether dice 1 or dice 2 will go first) -> conditional statement. Store the value in an array
+
+// 4. After both players have rolled and chosen dice order, the player with the higher combined number wins. -> to compare the 2 dice value
+
+// 5. The game will reset and will go back to Player 1
+
+//To define Global Variable
+var gameModeRollDice = "game mode roll dice";
+var gameModeChooseDice = "game mode choose dice";
+var gameModeDetermineWinner = "game mode determine winner";
+//start the gamemode to roll the dice
+var gameMode = gameModeRollDice;
+var currentPlayerRoll = [];
+//To track the current player along the game
+var currPlayer = 1;
+var allPlayerScore = [];
+
+//Roll dice helper function to return random value from 1 - 6
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+// Roll 2 dice helping function to return message
+var rollDiceForGame = function () {
+  var counter = 0;
+  while (counter < 2) {
+    currentPlayerRoll.push(rollDice());
+    counter = counter + 1;
+  }
+  console.log("rolldice for player", currentPlayerRoll);
+
+  return `🎲 Welcome Player ${currPlayer} 🎲 <br><br> You rolled ${currentPlayerRoll[0]} for Dice 1 and ${currentPlayerRoll[1]} for Dice 2 <br><br> Choose the order of the dice by entering the number "1" or "2".`;
+};
+
+var getPlayerScore = function (playerInput) {
+  var playerScore;
+  console.log("playerScore");
+  // to prompt error message if the input is not '1' or '2'
+  if (playerInput != 1 && playerInput != 2) {
+    return `Please only input either '1' or '2' to choose which dice to use as the first digit.`;
+  }
+  console.log("input==1");
```

Comment:
i would put this inside the if function and maybe write "Running in If condition, PlayerInput = 1"